sphinx/texinputs/sphinx.sty

```diff
@@ -539,12 +539,56 @@
 }
 
 % Notices / Admonitions
-%
 
+% Some are quite plain
+\newenvironment{sphinxlightbox}{%
+  \par\allowbreak
+  \noindent{\color{spx@notice@bordercolor}%
+            \rule{\linewidth}{\spx@notice@border}}\par\nobreak
+  {\parskip\z@skip\noindent}%
+  }
+  {%
+  \par
+  % counteract previous possible negative skip (French lists!):
+  % (we can't cancel that any earlier \vskip introduced a potential pagebreak)
+  \ifdim\lastskip<\z@\vskip-\lastskip\fi
+  \nobreak\vbox{\noindent\kern\@totalleftmargin
+      {\color{spx@notice@bordercolor}%
+       \rule[\dimexpr.4\baselineskip-\spx@notice@border\relax]
```

Comment:
test with 
```
.. Note:: This is a note

   with some contents


.. Note:: This is a note

   with some contents

.. Note:: This is a note

   with some contents
 ```

in image below, the left side is old rendering. The right side is new rendering, the bottom rule is 0.5pt lower. Top rules and text position not affected. I didn't initially add the `+0.5pt` because of bad habit to make life easier for computer... with the `0.5pt` rendering will be exactly as in old situation. In earlier code the bottom edge of bottom rule is positioned at `+0.4\baselineskip` of baseline. But this bad with very thick border. The new code positions the top edge of bottom rule at `+0.4\baselineskip` of baseline. As default thickness was `0.5pt` this means it is `0.5pt` lower in that case (if I don't add the `+0.5pt` in the `\dimexpr`).

![capture d ecran 2016-06-18 a 14 45 28](https://cloud.githubusercontent.com/assets/2589111/16170974/4ac89c42-3564-11e6-9e7a-ee3ef8196471.png)

The I captured smaller portion and zoomed it:
![capture d ecran 2016-06-18 a 14 47 07](https://cloud.githubusercontent.com/assets/2589111/16170977/5b9c40c8-3564-11e6-99c8-f79502687c33.png)